feature-libs/order/assets/translations/en/order.i18n.ts

```diff
@@ -14,7 +14,7 @@ export const order = {
     emptyPurchaseOrderId: 'None',
     none: 'None {{value}}',
     placed: 'Placed',
-    placedBy: 'Placed By',
+    placedBy: 'Buyer',
```

Comment:
This does not seem like a good idea as it might break existing usages. I'd rather add a new entry buyer with the 'Buyer' value. Having a key named placedBy and its value is 'Buyer; does not feel right.

Perhaps I'm missing something, in either case let me know.

Reply:
actually it is not used anywhere, but you are right - I updated